resources/views/includes/header.blade.php

```diff
@@ -61,7 +61,7 @@
         <li class="dropdown">
           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{{ Auth::user()->first_name }} <span class="caret"></span></a>
           <ul class="dropdown-menu">
-            <li><a href="{{ url('profile') }}">{{trans('app.profile.main')}}</a></li>
+            <li><a href="{{ url('admin/profile') }}">{{trans('app.profile.main')}}</a></li>
```

Comment:
Use `route` and work with named routes instead of `url`. Find the route that should replace this line `url('admin/profile')` in the route definitions.